tests/test_toy_examples.py

```diff
@@ -0,0 +1,7 @@
+import shlex
```

Comment:
this could be a doctest instead -- rather than an actual `tests/`

Reply:
Won't this required additional code that doctest runs to evaluate the example? I would think that we would just want to show the example JSON and not additional stuff, which is why I created the separate file and the pytest test in the PR.

Reply:
Yeah, I think I understand now. We should just include the JSON file from the `examples/` we already have -- but it'll only show latest schema version I suppose.

Reply:
> but it'll only show latest schema version I suppose

Yeah. I don't think this is _too_ big of a problem until/if we start releasing docs for different releases.